recipes/cpprestsdk/all/conanfile.py

```diff
@@ -57,8 +58,8 @@ def layout(self):
         cmake_layout(self, src_folder="src")
 
     def requirements(self):
-        self.requires("boost/1.83.0")
-        self.requires("openssl/[>=1.1 <4]")
+        self.requires("boost/[>=1.75.0 <2]")
+        self.requires("openssl/[>=1.1 <4]", transitive_headers=True)
```

Comment:
It would be nice to add a comment explaining why (in which public header it is included for example)